warehouse/models/mart/gtfs_quality/fct_daily_feed_guideline_checks.md

```diff
@@ -12,5 +12,6 @@ Here is a list of currently-implemented checks:
 | ------------------------------------ |---------|------------ |
 | No errors in MobilityData GTFS Schedule Validator |Compliance |GTFS Schedule Validator produced no errors for the transit provider’s static feed. |
 |No shapes-related errors appear in the MobilityData GTFS Validator | Accurate Service Data | None of the following shapes-related errors appear in the GTFS Schedule Validator: decreasing_shape_distance, equal_shape_distance_diff_coordinates, decreasing_or_equal_shape_distance, decreasing_or_equal_shape_distance |
+| Every trip in trips.txt has a shape_id listed | Accurate Service Data | Every trip in trips.txt has a shape_id listed.|
```

Comment:
Think this is an incorrect copy-paste?

Reply:
I meant to add this - forgot to update it for a previous check. But good catch, because in so doing I forgot to add the description for this check!!